src/store/actions/contact.actions.ts

```diff
@@ -50,6 +54,29 @@ export const createContact = (args: ICreateContactArgs): ThunkAction<Promise<voi
   };
 };
 
+export const updateContact = (args: IUpdateContactArgs): ThunkAction<Promise<void>, RootState, unknown, Action> => {
+  return async (dispatch: ThunkDispatch<RootState, unknown, Action>) => {
+    dispatch({type: CONTACTS_LOADING});
+    editContact(args)
+    .then(()=> {
```

Comment:
please refactor this  to a promise chain that is not nested with only 1 catch at the end. this is unreadable

Reply:
made to return single contact